src/components/expression/expression.tsx

```diff
@@ -123,18 +124,30 @@ export const EuiExpression: React.FunctionComponent<
       'euiExpression-isUppercase': uppercase,
     },
     displayToClassNameMap[display],
-    colorClass
+    colorToClassNameMap[calculatedColor]
   );
 
   const Component = onClick ? 'button' : 'span';
 
+  const descriptionStyle = descriptionProps && descriptionProps.style;
   const customWidth =
     display === 'columns' && descriptionWidth
       ? {
-          flexBasis: `${descriptionWidth}%`,
+          flexBasis: descriptionWidth,
+          ...descriptionStyle,
         }
       : undefined;
```

Comment:
Need to also account for the fact that `style` is a property of `descriptionProps`. If a consumer passed `descriptionProps.style` it would have overridden the `customWidth` style.